### ISSUE TITLE
[BUG]: Error transform sourcePath making screenshot not shown in reports

### ISSUE THREAD

**Issue by jesmartorres24** [healenium/healenium-web#239]
### Describe the bug

Error transform sourcePath making screenshot not shown in reports

### How to reproduce the issue

```shell
- Setup postgres healenium account
- created a test script to test healing
- it healed the element
- it created a screenshot folder
- opened reports page and src didn't edit the absolute path.
```

### Logs appeared during using Healenium

```shell
[Build Report] Error transform sourcePath: C:\Users\{user\path}\screenshots\83c74b400b362e1652c048fc73cce5ef\screenshot_30-Jun-2023-03-37-10.616.png
WARN 23832 - [7979-exec-5] healenium : [Build Report] Error transform sourcePath: C:\Users\{user\path}\screenshots\83c74b400b362e1652c048fc73cce5ef\screenshot_30-Jun-2023-03-37-10.616.png
2023-07-05 08:31:16.468  WARN 23832 - [7979-exec-2] healenium : [Build Report] Error transform sourcePath: C:\Users{user\path}\screenshots\83c74b400b362e1652c048fc73cce5ef\screenshot_30-Jun-2023-03-37-10.616.png

edited the absolute path for confidentiality
```

### Expected behavior

Screenshot will appear in reports page

### Actual behavior

It doesn't load

### Healenium Web version

1.3.4

### Healenium Backend version

3.3.4

### Selenium version

4.9.1

### Platform

Java

### Properties file

recovery-tries = 1
score-cap = 0.5
heal-enabled = true
hlm.server.url = http://localhost:7979
hlm.imitator.url = http://localhost:8000

### Additional context

This is what it looks like when I load reports page. 
![image](https://github.com/healenium/healenium-web/assets/104480296/0b303591-af93-4f98-9464-a8d62c4222f4)

changed the src and removed the absolute path to start with /screenshots/
![image](https://github.com/healenium/healenium-web/assets/104480296/f8fcf237-e64d-4597-9d3d-0d12d114a9f9)

This is what the logs show when I try to load the image.
![image](https://github.com/healenium/healenium-web/assets/104480296/b1c40540-8af0-4f58-bd58-db73d21db082)

**Comment by Alex-Reif** [comment]
Hi @jesmartorres24 , We will fix it in next release.

**Comment by Alex-Reif** [comment]
Hi @jesmartorres24 , Issue fixed in the hlm-backend 3.4.0. Update service, please.

**Comment by jesmartorres24** [comment]
Hi @Alex-Reif, thank you so much. It's now working properly. :)
